iGCS.xcodeproj/project.pbxproj

```diff
@@ -337,13 +343,10 @@
 		CA26AD6B173E22E300F8E7B3 /* MissionItemField.m in Sources */ = {isa = PBXBuildFile; fileRef = CA26AD6A173E22E300F8E7B3 /* MissionItemField.m */; };
 		CA2D60161A733FD800AA2CBB /* GCSCraftModelGenerator.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D60151A733FD800AA2CBB /* GCSCraftModelGenerator.m */; };
 		CA2D60171A733FD800AA2CBB /* GCSCraftModelGenerator.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D60151A733FD800AA2CBB /* GCSCraftModelGenerator.m */; };
-		CA2D60181A733FD800AA2CBB /* GCSCraftModelGenerator.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D60151A733FD800AA2CBB /* GCSCraftModelGenerator.m */; };
 		CA2D601D1A7342BD00AA2CBB /* GCSCraftArduCopter.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D601C1A7342BD00AA2CBB /* GCSCraftArduCopter.m */; };
 		CA2D601E1A7342BD00AA2CBB /* GCSCraftArduCopter.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D601C1A7342BD00AA2CBB /* GCSCraftArduCopter.m */; };
-		CA2D601F1A7342BD00AA2CBB /* GCSCraftArduCopter.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D601C1A7342BD00AA2CBB /* GCSCraftArduCopter.m */; };
 		CA2D60221A734FE600AA2CBB /* GCSCraftArduPlane.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D60211A734FE600AA2CBB /* GCSCraftArduPlane.m */; };
 		CA2D60231A734FE600AA2CBB /* GCSCraftArduPlane.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D60211A734FE600AA2CBB /* GCSCraftArduPlane.m */; };
-		CA2D60241A734FE600AA2CBB /* GCSCraftArduPlane.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D60211A734FE600AA2CBB /* GCSCraftArduPlane.m */; };
```

Comment:
Yes having them in the test target keeps it from compiling.



On Jan 27, 2015, at 6:45 PM, Claudio Natoli <notifications@github.com>
wrote:

In iGCS.xcodeproj/project.pbxproj
<https://github.com/fightingwalrus/iGCS/pull/164#discussion_r23662429>:

>  		CA2D60221A734FE600AA2CBB /* GCSCraftArduPlane.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D60211A734FE600AA2CBB /* GCSCraftArduPlane.m */; };
>  		CA2D60231A734FE600AA2CBB /* GCSCraftArduPlane.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D60211A734FE600AA2CBB /* GCSCraftArduPlane.m */; };
> -		CA2D60241A734FE600AA2CBB /* GCSCraftArduPlane.m in Sources */ = {isa = PBXBuildFile; fileRef = CA2D60211A734FE600AA2CBB /* GCSCraftArduPlane.m */; };

Few files dropped from one of the targets - intentional?

—
Reply to this email directly or view it on GitHub
<https://github.com/fightingwalrus/iGCS/pull/164/files#r23662429>.